docs/README.md

```
@@ -0,0 +1,174 @@
+
+[![N|Solid](static/img/markdown/misc/logo.png)](https://github.com/PalisadoesFoundation/talawa-api)
+
+# Installation
+
+This document provides instructions on how to set up and start a running instance of the [talawa-api documentation website](https://docs-api.talawa.io/) on your local system. The instructions are written to be followed in sequence so make sure to go through each of them step by step without skipping any sections.
+
+# Table of Contents
+
+<!-- /TOC -->
+
+- [Developer-Docs Installation](#talawa-api-installation)
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)